front/src/core/field.validation.ts

```diff
@@ -0,0 +1,13 @@
+import { ValidationSchema, Validators } from '@lemoncode/fonk';
+import { createFormikValidation } from '@lemoncode/fonk-formik';
+
+const validationSchema: ValidationSchema = {
```

Comment:
This the file that should be on every pod, the ValidationSchema

Is specific per pod